doc/compile_Windows.md

```diff
@@ -75,10 +75,11 @@
 
 ## Compile
 
-- download and unzip `xmr-stak`
+- download `xmr-stak` [Source Code.zip](https://github.com/fireice-uk/xmr-stak/releases) and save to a location in your `User` folder
+- extract the Source Code.zip file
 - open the command line terminal `cmd`
-- `cd` to your unzipped source code directory
-- execute the following commands (NOTE: path to VS2017 can be different)
+- `cd` to the extracted source code directory
+- execute the following commands (NOTE: path to Visual Studio 2017 Community can be different)
   ```
   "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsMSBuildCmd.bat"
   set CMAKE_PREFIX_PATH=C:\xmr-stak-dep\hwloc;C:\xmr-stak-dep\libmicrohttpd;C:\xmr-stak-dep\openssl
```

Comment:
Might want to mention something about how the next step, `mkdir build` + `cd build` can silently and deviously find you auto-routed inside a `C:\Users\USERNAME\source\build\` instead of the extracted xmr-stak dir you thought you were in. Thanks, Windows. Switching back into the correct dir just before making your `mkdir build` fixes the "There are no makefiles inside this directory, duuude" error.

Reply:
I agree. Added more detailed instruction point there.